anycable-go/templates/deployment.yml

```diff
@@ -161,9 +161,17 @@ spec:
          {{- end }}
         terminationMessagePath: /dev/termination-log
         terminationMessagePolicy: File
+        volumeMounts:
```

Comment:
we should plan on providing a patch to the upstream repository, so this can be configurable.
@adimania Can you create a story for this improvement?